### ISSUE TITLE
Issue with condition designer using PGSQL

### ISSUE THREAD

**Issue by 1bigchonk** [martialblog/docker-limesurvey#24]
When trying to use condition designer on a survey, limesurvey throws a database error. I've since switched to mysql as the database and condition designer works. Associated error: 

```
2020-02-27 14:47:49.037 UTC [12655] ERROR:  missing FROM-clause entry for table "questiongroupl10ns" at character 1354
2020-02-27 14:47:49.037 UTC [12655] STATEMENT:  SELECT "t"."qid" AS "t0_c0", "t"."parent_qid" AS "t0_c1", "t"."sid" AS "t0_c2", "t"."gid" AS "t0_c3", "t"."type" AS "t0_c4", "t"."title" AS "t0_c5", "t"."preg" AS "t0_c6", "t"."other" AS "t0_c7", "t"."mandatory" AS "t0_c8", "t"."encrypted" AS "t0_c9", "t"."question_order" AS "t0_c10", "t"."scale_id" AS "t0_c11", "t"."same_default" AS "t0_c12", "t"."relevance" AS "t0_c13", "t"."modulename" AS "t0_c14", "group"."gid" AS "t1_c0", "group"."sid" AS "t1_c1", "group"."group_order" AS "t1_c2", "group"."randomization_group" AS "t1_c3", "group"."grelevance" AS "t1_c4", "questionGroupL10ns"."id" AS "t2_c0", "questionGroupL10ns"."gid" AS "t2_c1", "questionGroupL10ns"."group_name" AS "t2_c2", "questionGroupL10ns"."description" AS "t2_c3", "questionGroupL10ns"."language" AS "t2_c4", "questionL10ns"."id" AS "t3_c0", "questionL10ns"."qid" AS "t3_c1", "questionL10ns"."question" AS "t3_c2", "questionL10ns"."help" AS "t3_c3", "questionL10ns"."script" AS "t3_c4", "questionL10ns"."language" AS "t3_c5" FROM "lime_questions" "t"  LEFT OUTER JOIN "lime_groups" "group" ON ("t"."gid"="group"."gid") LEFT OUTER JOIN "lime_group_l10ns" "questionGroupL10ns" ON ("questionGroupL10ns"."gid"="group"."gid") LEFT OUTER JOIN "lime_question_l10ns" "questionL10ns" ON ("questionL10ns"."qid"="t"."qid") WHERE ("t"."qid"=71 AND "t"."parent_qid"=0 AND "t"."sid"=348556) AND (questionGroupL10ns.language = 'en')

```

**Comment by martialblog** [comment]
Do you happen to know that the input here was? `character 1354`

Looks like an application error to me

**Comment by martialblog** [comment]
Hi, so I could recreate that with docker-compose

I simply started a new instance with Postgres, created a new Survey and tried to open the Question List. 

`
ERROR:  syntax error at or near "group" at character 981
SELECT "t"."qid" AS "t0_c0", "t"."parent_qid" AS "t0_c1", "t"."sid" AS "t0_c2", "t"."gid" AS "t0_c3", "t"."type" AS "t0_c4", "t"."title" AS "t0_c5", "t"."preg" AS "t0_c6", "t"."other" AS "t0_c7", "t"."mandatory" AS "t0_c8", "t"."encrypted" AS "t0_c9", "t"."question_order" AS "t0_c10", "t"."scale_id" AS "t0_c11", "t"."same_default" AS "t0_c12", "t"."relevance" AS "t0_c13", "t"."modulename" AS "t0_c14", "group"."gid" AS "t1_c0", "group"."sid" AS "t1_c1", "group"."group_order" AS "t1_c2", "group"."randomization_group" AS "t1_c3", "group"."grelevance" AS "t1_c4", "ql10n"."id" AS "t2_c0", "ql10n"."qid" AS "t2_c1", "ql10n"."question" AS "t2_c2", "ql10n"."help" AS "t2_c3", "ql10n"."script" AS "t2_c4", "ql10n"."language" AS "t2_c5" FROM "lime_questions" "t"  LEFT OUTER JOIN "lime_groups" "group" ON ("t"."gid"="group"."gid") LEFT OUTER JOIN "lime_question_l10ns" "ql10n" ON ("ql10n"."qid"="t"."qid") WHERE ((t.sid='784458') AND (t.parent_qid='0')) AND (language='en') ORDER BY group.group_order asc, t.question_order asc LIMIT 10
`

I still think that's an application error. I will open an issue with the LimeSurvey Devs

**Comment by martialblog** [comment]
I opened this issue in the LImeSurvey Bugtracker: https://bugs.limesurvey.org/view.php?id=15961

**Comment by martialblog** [comment]
Issue is fixed in latest version 4.1.9+200310
